README.md

```diff
@@ -6,10 +6,10 @@ The challenge is to have your name added to the list below.<br/>
 ## You don't know git and you wanna learn it?
 Take a look at this list of [Git Learning Material](LearningResources.md).
 
-### If you already know git and need a refresher/cheatsheet
+### If you already know git and need a refresher/cheatsheet:
 Take a look at [this cheatsheet](Cheatsheet.md)
 
-### To complete the challenge do the following:
+### - To complete the challenge do the following:
```

Comment:
What about removing the dash from here to be consistent with all of the other headers ?

Reply:
I made just to differentiate the sentence to make OSCians to know where to "complete the challenge" .. but it's ok if you remove it 😄